README.md

```diff
@@ -1 +1,56 @@
 # discord-r9k
+
+An experimental [Robot 9000](https://blog.xkcd.com/2008/01/14/robot9000-and-xkcd-signal-attacking-noise-in-chat/) for Discord.
+
+## Usage
+
+First, o to the [Discord developer documentation](https://discordapp.com/developers/), create an app and get your token.
+
+```
+export DISCORD_TOKEN=<token>
+npm install
+node index.js
+```
+
+To add the bot to your server, insert the client ID into this URL https://discordapp.com/oauth2/authorize?client_id=CLIENT_ID_GOES_HERE&scope=bot&permissions=0 and paste it into your browser of choice.
+
+The bot as it is currently has a lot of things tuned to my personal discord server, which I plan to refactor away at some point, in the mean time, you will probably need to head into the code and take these out if they end up annoying you.
+
+## Documentation
+
+### `addMessage(trigger, callback)`
+
+#### `tirgger`
+
+A String, RegExp or Array, that will trigger this event.
```

Comment:
I would change this to list the valid types e.g.

The trigger can be one of the following types:
* **String** - The callback will be triggered if the message matches the String exactly
* **Regular Expression** - if the message matches the RegExp, the callback will be triggered, the callback argument will include the list of matches returned by [`String.match()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match)
* **Array** -  All array elements are treated as separate triggers.

Reply:
Also `s/seperate/separate/`